src/hooks/useBankTransactions/useBankTransactions.tsx

```diff
@@ -57,18 +51,52 @@ export const useBankTransactions: UseBankTransactions = params => {
       return [false, undefined]
     }
 
-    if (!prevData?.meta?.pagination?.cursor) {
+    if (_index === 0) {
```

Comment:
NIT:  Underscore can be removed from `_index` here:
 
```suggestion
    if (index === 0) {
```

... and in line 49:

```ts
const getKey = (index: number, prevData: any) => {
```